src/main/java/com/parasoft/findings/utils/common/util/FileUtil.java

```diff
@@ -297,8 +297,8 @@ private static void close(InputStream is)
         }
         try {
             is.close();
-        } catch (IOException exc) { // parasoft-suppress SECURITY.UEHL.LGE "acceptable"
-            // do nothing
+        } catch (IOException exc) { // parasoft-suppress SECURITY.UEHL.LGE "acceptable" // parasoft-suppress OWASP2021.A9.LGE "This is intentionally designed to ensure exceptions during input stream closing don't cause the process to fail."
+            Logger.getLogger().error("An exception is thrown during closing output stream.", exc);
```

Comment:
Instead of using the term "An exception is thrown," use something like "Failed to close the output stream."
Same for line 617 and line 322 and `URLUtil.java`